app/controllers/forest_liana/base_controller.rb

```diff
@@ -4,6 +4,10 @@ class BaseController < ::ActionController::Base
     wrap_parameters false
     before_action :reject_unauthorized_ip
 
+    def route_not_found
+      head :not_found
+    end
```

Comment:
It was in Application controller before, I think it makes more sense to put it in the BaseController since here we don't run the authentication middleware before the controller's methods are called. (If a route is not found we don't need to check if the user is authenticated).